src/styles/theme.js

```diff
@@ -30,6 +30,10 @@ const monochrome = {
 const rgba = {
   doveGray: 'rgb(107, 101, 101,0.5)',
   dialogGray: 'rgba(0, 0, 0, 0.25)',
+  silverTree: 'rgba(112, 189, 158, 0.40)',
+  finn: 'rgba(90, 47, 86, 0.3)',
+  greyHover: 'rgba(222, 213, 221, 0.8)',
```

Comment:
Since greyHover is used in the app and I can't see that view import of this color has changed path it will break

Reply:
And you can remove silverTree and finn instead of moving them, as they are not used in the app

Reply:
it won't break as you do `...rgba` in `colors` object

Reply:
corrected